skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
```

Comment:
@vighneshbirodkar 
Yes, you are right.
MBLBP is straightforward, but reading OpenCV trained file and check
if it works is not. This is what I want to test.
After this I will be dead-sure about everything and will optimize everything.
Is it okay ?